schema/spacewalk/upgrade/susemanager-schema-4.3.9-to-susemanager-schema-4.3.10/011-rhnActionDup-add-missing_successor.sql

```diff
@@ -0,0 +1,3 @@
+-- add the new column to store the products which have no successors
+
+ALTER TABLE rhnActionDup ADD COLUMN IF NOT EXISTS missing_successors VARCHAR(512);
```

Comment:
If this gets not merged for Beta 3, you need to change the directory.

Reply:
@admd IMHO yes: this is a big PR. +433 −99, and changes 3 key packages (java, schema, sls)

If we merge it now, it could break more tests, and FMPOV the priorities are now:
- Getting a stable testsuite ASAP
- Merge only blockers

@mcalmer opinion?